server/helper/ImageUtils.js

```diff
@@ -12,13 +27,23 @@ const THUMBNAIL_POSTFIX = "-thumb";
  * @param {*} path The path of the image to which a thumbnail should be created.
  */
 async function createThumbnail(path, targetHeight) {
```

Comment:
die create thumbnail funktion brauchen wir jetzt doch eigentlich garnicht mehr, oder?
das thumb wird ja in createSmallerSizes schon erstellt. die könnten wir doch für alle anderen Bilder auch benutzen, oder?